server/graphql/v1/mutations.js

```diff
@@ -208,6 +208,18 @@ const mutations = {
       return updateUserEmail(remoteUser, email);
     },
   },
+  setChangelogViewDate: {
+    type: UserType,
+    description: 'Update the time which the user viewed the changelog updates',
+    args: {
+      changelogViewDate: {
+        type: new GraphQLNonNull(GraphQLString),
```

Comment:
To provide better validations

```suggestion
        type: new GraphQLNonNull(IsoDateString),
```